src/Modules/OrchardCore.Commerce.Payment/Services/PaymentService.cs

```diff
@@ -267,7 +267,7 @@ public async Task<IList<string>> UpdateOrderWithDriversAsync(ContentItem order)
     public async Task UpdateOrderToOrderedAsync(
         ContentItem order,
         string? shoppingCartId,
-        Func<OrderPart, IEnumerable<IPayment>?>? getCharges = null)
+        Func<OrderPart, IEnumerable<Commerce.Abstractions.Models.Payment>?>? getCharges = null)
```

Comment:
you don't seem to like using lol

Reply:
The IDE automatically adds as much `using` as possible. I also use <kbd>Crtl</kbd>+<kbd>R</kbd>, <kbd>Crtl</kbd>+<kbd>G</kbd> all the time to automate applying `using`s. Every case you highlighted above can't be made shorter.